pyteal/ast/arg.py

```diff
@@ -1,13 +1,17 @@
 from ..types import TealType
+from ..ir import TealOp, Op
 from ..errors import TealInputError
 from .leafexpr import LeafExpr
 
 class Arg(LeafExpr):
-    """An expression to get an argument."""
-    
-    def __init__(self, index:int) -> None:
+    """An expression to get an argument when running in signature verification mode."""
```

Comment:
I wonder if it makes sense to add mode checks statically.

Reply:
Yes, the `verifyOpsForMode` function in `compiler.py` will raise an error if an unsupported op is used. The docstring is purely to help direct users to the correct expression to use based on the mode they want.